### ISSUE TITLE
julia does not build

### ISSUE THREAD

**Issue by idontgetoutmuch** [thomasjm/julia2nix#8]
`nix-build '<nixpkgs>' -A julia`

gives

```
ld: library not found for -lgfortblas
clang-7: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by idontgetoutmuch** [comment]
Sorry I should probably report this to nix.

**Comment by thomasjm** [comment]
Yep, getting this working is a prerequisite. Note that Julia 1.3 is the latest release available on the latest stable Nix release 20.09.
